### PR TITLE
Pixel shifting fix

### PR DESCRIPTION
**Pull request by ReddicusDragon**
AIs could pixel shift. Blob camera could pixel shift. No more, though. Through a generous application of spaghetti code, I have fixed monster's buggy pixel shifting. **It Just Works™**

Oh, and it also makes pixel shifting reset on player move.

:cl:  
rscadd: Added pixel shifting auto reset on player mob movement
rscdel: Removed AI pixel shifting
/:cl:

### CODE REVIEW

File: code/modules/mob/mob_movement.dm [hunk 1]
```diff
@@ -75,7 +75,11 @@
 
 	if(!mob.Process_Spacemove(direct))
 		return FALSE
-
+	if (!src.prefs.bindings.isheld_key("Ctrl"))
```
Comment (Bibby0110) [inline code/modules/mob/mob_movement.dm:78]: `if (!src.prefs.bindings.isheld_key("Ctrl") && !(mob.datum_flags & DF_VAR_EDITED))`
i want pixel shifted Ian thank you very much

**Review by Bibby0110 — COMMENTED** [review]
uwu

File: code/modules/mob/mob_movement.dm (hunk 1 above)
Comment (monster860) [reply code/modules/mob/mob_movement.dm:78]: thats shit code gamer

Comment (Bibby0110) [reply code/modules/mob/mob_movement.dm:78]: nah

**Comment by azzzertyy** [comment]
why is either of these pixel shifting a bad thing?

**Comment by ReddicusDragon** [comment]
@azzzertyy basically pixel shifting not auto resetting gives a combat advantage and ais shouldn’t be able to pixelshift because that’s just shit code

**Comment by azzzertyy** [comment]
yeah but the hitboxes are exactly the same with these, no ones meleeing a blob or ai, and if so, moving your mouse to the top right corner of the pixel isnt very difficult

**Comment by Hopekz** [comment]
FYI; It was specifically made not to reset VS the citadel port.

**Comment by ReddicusDragon** [comment]
@monster860 opinion?

**Review by monster860 — CHANGES_REQUESTED** [review]
Can you just go to the Citadel version and copy the code from there, as that one will only reset the pixel_x/y if actual pixel shifting was done, unlike this one which is aways resetting the pixel_x, and not necessarily to the correct value

**Pull request closed without merging**